traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -142,7 +141,8 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.4, http.MethodGet, `cdns/dnsseckeys/refresh/?(\.json)?$`, cdn.RefreshDNSSECKeys, auth.PrivLevelOperations, Authenticated, nil},
 
 		//CDN: Monitoring: Traffic Monitor
-		{1.1, http.MethodGet, `cdns/{cdn}/configs/monitoring(\.json)?$`, monitoring.Get, auth.PrivLevelReadOnly, Authenticated, nil},
+		{1.1, http.MethodGet, `cdns/{cdn}/configs/monitoring(\.json)?$`, crconfig.SnapshotGetMonitoringHandler, auth.PrivLevelReadOnly, Authenticated, nil},
+		{1.2, http.MethodGet, `cdns/{name}/configs/monitoring(\.json)?$`, crconfig.SnapshotGetMonitoringHandler, auth.PrivLevelReadOnly, Authenticated, nil},
```

Comment:
Actually, the second one is both broken, and will never get called. Routes are called in the order they're registered. So, a 1.2 route has to be before a 1.1 route with the same path, or it'll never get called. But it's broken anyway, because it's using `{name}`, but the handler itself expected a param named `{cdn}`. So yeah, needs removed.

Reply:
Ok, I wasnt sure how those worked completely.  Previously there were actually 2 definitions for the monitoring.json but in separate locations in the routes.go. I figured it was time to consolidate them at this point since that was just confusing but I can just move it all to the single one